### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by kulynychruslana**
- [DEMO LINK](https://kulynychruslana.github.io/layout_stars/)
- [TEST REPORT LINK](https://kulynychruslana.github.io/layout_stars/report/html_report/)

### CODE REVIEW

**Review by mykhalenych — CHANGES_REQUESTED** [review]
check comments, also your demo and test links aren't working

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  width: 16px;
+  height: 16px;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1) {
```
Comment (mykhalenych) [inline src/style.css:22]:
```suggestion
.stars--1 :nth-child(1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
... {
  background-image: url(images/star-active.svg);
  }
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
```
Comment (Esceype) [inline src/style.css:40]: Checklist 2nd point

**Review by Esceype — CHANGES_REQUESTED** [review]
Read the checklist, at all good job

File: src/index.html [hunk 3]
```diff
@@ -8,6 +8,49 @@
...
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
```
Comment (denys-danyliuk) [inline src/index.html:25]:
```suggestion
    <div class="container">
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
      
      <div class="stars stars--1">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```
1'st point in checklist

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1) ,
+
+.stars--2 :nth-child(-n + 2),
```
Comment (denys-danyliuk) [inline src/style.css:24]:
```suggestion
.stars--1 :nth-child(1) ,
.stars--2 :nth-child(-n + 2),
```

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**